docs/explanations/flyscanning.md

```diff
@@ -0,0 +1,86 @@
+
+* Fly scanning: Short for "on-the-fly scanning," this method involves continuous motion during data acquisition. The motion and data collection processes
+happen simultaneously, enabling faster scans and smoother coverage of the scanned domain. Devices capable of this mode implement the [Flyable](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.Flyable) and [Collectable](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.Collectable) protocols.
+
+Filewriting detectors additionally implement [WritesStreamAssets](https://blueskyproject.io/bluesky/main/hardware.html#bluesky.protocols.WritesStreamAssets) to report the data written to disk during the scan, either at each data point in a step scan, or periodically during a flyscan.
+
+## Scan frames and collection windows
+
+Consider some trajectory in 2D space:
+
+![A flyscan trajectory with collection windows containing frames](../images/flyscan_collection_windows_and_frames.svg)
+
+Along the flyscan trajectory we define discrete collection windows, which are intervals where a coherent block of data is acquired.
+The data collection block is the scan frame, defined as [a vector of lower, mid and upper points in scan space](https://blueskyproject.io/scanspec/main/explanations/technical-terms.html#frame). Note that a step scan could also be understood as a flyscan where the frames' lower and upper bounds are the same as their midpoints.
+Scan frames map directly to Bluesky [Events](https://blueskyproject.io/bluesky/main/documents.html#event), and typically corresponds to a single detector exposure, though they may be composed of multiple detector frames.
+
+## Synchronisation mechanisms
+
+A flyscan must define the start of each collection window and the acquisition of scan frames within those. Various mechanisms can be used, depending on the hardware and experimental requirements.
+
+### Collection window triggers
+
+Collection windows are level-triggered, meaning the window is open while the level is high.
+
+* Software trigger: The control software signals the start of the next window. For example, a plan kicks off a linear motor move, and starts a free-running detector when it expects the motor to be at constant velocity.
+* Time-based trigger: A window opens at predefined time intervals (this could be a software or a hardware trigger e.g. from a PandA box).
+* Readback trigger: A hardware trigger sent when a signal from a sensor or encoder value is positively evaluated through some position compare logic.
+* Setpoint trigger: The same control signal sent to some device (e.g. a sample environment controller) also triggering the start of the collection window.
+
+### Scan frame triggers
+
+Scan frames are edge-triggered, meaning that only the rising edge of the trigger is registered.
```

Comment:
Again, the internal triggers are not edge or level triggered. Hardware triggers sent to the detector can be either edge or level triggered though...